src/plugins/info/info_impl.cpp

```diff
@@ -167,63 +157,63 @@ void InfoImpl::process_flight_information(const mavlink_message_t& message)
     _flight_information_received = true;
 }
 
-void InfoImpl::translate_binary_to_str(
-    uint8_t* binary, unsigned binary_len, char* str, unsigned str_len)
+std::string InfoImpl::translate_binary_to_str(uint8_t* binary, unsigned binary_len)
 {
+    std::string str(binary_len * 2, '0');
```

Comment:
@julianoes: Does this length make sense to you? Seems to be working, but... :laughing:

Reply:
So `binary_len` is the number of bytes from `sizeof` but doesn't each byte need 2 hex chars?